Produlytics/ui/back-end/src/main/java/it/deltax/produlytics/uibackend/admins/devices/business/CharacteristicConstraints.java

```diff
@@ -11,12 +11,9 @@ public interface CharacteristicConstraints {
    * @return true se e solo se le informazioni rispettano i vincoli
    */
   static boolean characteristicConstraintsOk(CharacteristicConstraintsToCheck toCheck) {
-    final boolean limitsOk =
-        toCheck.autoAdjust()
-            || (toCheck.upperLimit().isPresent() && toCheck.lowerLimit().isPresent());
+    final boolean autoAdjust = toCheck.autoAdjust();
+    final boolean technicalLimits = toCheck.upperLimit().isPresent() && toCheck.lowerLimit().isPresent();
 
-    final boolean sampleSizeOk = !toCheck.autoAdjust() || toCheck.sampleSize().isPresent();
-
-    return limitsOk && sampleSizeOk;
+    return autoAdjust || technicalLimits;
```

Comment:
Non mi ricordo com'era il discorso. Se sample-size è `null` e serve viene considerato 0 in automatico?

Reply:
Forse avrei dovuto spiegarmi meglio.

Praticamente il problema era che se `sampleSize` viene settato a 0 invece che a null allora non vengono selezionate le rilevazioni per calcolare media e varianza. Questa PR dovrebbe sistemare quel problema permettendo a `sampleSize` di essere null... Ma ora che ci penso che l'utente mette 0 manualmente si romperebbe tutto comunque...

Reply:
Vabbè comunque questa cosa sistemi tu? Io intanto metto a posto i test.